aspnetcore/signalr/version-differences.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Differences between SignalR and SignalR Core
+author: rachelappel
+description: Differences between SignalR and SignalR Core
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/version-differences
+---
+
+# Differences between SignalR and ASP.NET Core SignalR
+
+ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+
+## Feature differences
+
+### Automatic reconnects
+
+Automatic reconnects are no longer supported. Previously, SignalR tried to reconnect to the server if the connection was dropped. Now, if the client is disconnected the user must explicitly start a new connection if they want to reconnect.
+
+### Protocol support
+
+ASP.NET Core SignalR supports JSON, binary, text, and custom protocols. [MessagePack](xref:signalr/messagepackhubprotocol) is a binary serialization format that is fast and compact.
+
+### Use SignalR with Websockets
+
+SignalR now supports using [WebSockets](xref:fundamentals/websockets) to work directly with a socket connection.
+
+## Differences on the server
+
+The SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
+
+SignalR is an ASP.NET Core middleware, so it must be configured in the `Startup` class by calling `AddSignalR`.
+
+```csharp
+services.AddSignalR();
```

Comment:
I'd suggest separating these a little bit into two snippets and call out that `AddSignalR` goes in to `ConfigureServices` and `UseSignalR` goes in to `Configure`